crates/iroha_torii/src/lib.rs

```diff
@@ -137,13 +151,6 @@ impl Torii {
                     let metrics_reporter = self.metrics_reporter.clone();
                     move || core::future::ready(routing::handle_metrics(&metrics_reporter))
                 }),
-            )
-            .route(
-                uri::API_VERSION,
```

Comment:
I don't see a reason why this should be guarded behind telemetry. I'm willing to restore though